.github/workflows/testing.yml

```diff
@@ -0,0 +1,22 @@
+name: Dart
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: dart-lang/setup-dart@v1.3
+
+      - name: Install dependencies
+        run: dart pub get
+
+      - name: Run tests
+        run: dart test
```

Comment:
this should be an integration testing?

Mh! I think in there we do to do like this one https://github.com/dart-lightning/lndart.cln/blob/main/.github/workflows/testing_rpc.yml

But I will do it because it required to know how to run docker and how integrate it with CI, but for now it is a good starting point